test/tests-jsx.js

```diff
@@ -3634,10 +3634,78 @@ var fbTestFixture = {
 if (typeof exports !== "undefined") {
   var test = require("./driver.js").test;
   var testFail = require("./driver.js").testFail;
+  var tokTypes = require("../").tokTypes;
 }
 
 testFail("var x = <div>one</div><div>two</div>;", "Adjacent JSX elements must be wrapped in an enclosing tag (1:22)");
 
+test('<a>{/* foo */}</a>', {}, {
+  ranges: true,
```

Comment:
`ranges` and `loose` are not needed in this case - first one checks `range` props and second marks test as such that shouldn't be checked against loose parser - which JSX doesn't have anyway.